test/everypolitician_test.rb

```diff
@@ -195,4 +195,13 @@ def test_expose_the_statement_count_of_a_legislature
     end
   end
 
+  def test_lastmod_is_a_time
+    VCR.use_cassette('countries_json') do
+      australian_senate = Everypolitician.country(slug: 'Australia').legislature(slug: 'Senate')
+      assert_equal 2016, australian_senate.lastmod.year
+      assert_equal 7, australian_senate.lastmod.month
+      assert_equal 24, australian_senate.lastmod.day
+    end
+  end
+
```

Comment:
Aaaaahhh such a good point on the hour, yeah. Let me add that and the changelog.